tests/e2e/backup_restore_suite_test.go

```diff
@@ -139,14 +156,24 @@ var _ = Describe("AWS backup restore tests", func() {
 				log.Printf("Waiting for restic pods to be running")
 				Eventually(AreResticPodsRunning(namespace), timeoutMultiplier*time.Minute*3, time.Second*5).Should(BeTrue())
 			}
-			if brCase.BackupRestoreType == CSI {
+			if brCase.BackupRestoreType == CSI || brCase.BackupRestoreType == CSIDataMover {
```

Comment:
Not sure if this is relevant here, but data mover currently only supports aws and azure

Reply:
Thanks.

Reply:
resolved via 2704f28...a51f67d